Rakefile

```diff
@@ -13,44 +16,53 @@ require File.expand_path('../app/models/load_balancer_registration.rb', __FILE__
 
 ScalarmStorageManager::Application.load_tasks
 
-namespace :log_bank do
-  desc 'Start the service'
-  task :start => :environment do
-    %x[thin start -d -C config/thin.yml]
-  end
-
-  desc 'Stop the service'
-  task :stop => :environment do
-    %x[thin stop -C config/thin.yml]
-  end
-end
+# TODO: db_instance_dbpath should not be relative to DB_BIN_PATH
+# because it makes difficult to use external mongo if available
 
 # configuration - path to a folder with database binaries
 DB_BIN_PATH = File.join('.', 'mongodb', 'bin')
 LOCAL_IP = UDPSocket.open {|s| begin s.connect("64.233.187.99", 1); s.addr.last rescue "127.0.0.1" end }
 
 namespace :service do
-  task :start => [:environment, 'db_instance:start', 'db_config_service:start', 'log_bank:start' ] do
+  task :start => [:environment, 'service:ensure_config', 'db_instance:start', 'db_config_service:start', 'log_bank:start' ] do
     load_balancer_registration
   end
 
-  task :stop => [:environment, 'log_bank:stop', 'db_config_service:stop', 'db_instance:stop' ] do
-    load_balancer_deregistration
+  task :stop => [:environment, 'log_bank:start', 'db_config_service:start', 'db_instance:start' ] do
```

Comment:
Otworzyłem wszystkie pull reqeusty i sobie przeglądałem. Nie zauważyłem że to jakoś strasznie stare. Złota łopata dla mnie.